### PR TITLE
Change category

### PR DESCRIPTION
**Pull request by amyria3**
The point must receive:
  item_id
  target_category_id
as request-body.
It will then pick then update the  customCategoryId of Item

### CODE REVIEW

**Review by NoHara42 — CHANGES_REQUESTED** [review]
see small change

File: src/pages/api/changeCategory.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { z, ZodError } from "zod";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
```
[the remark this answers is not in the document]
Comment (amyria3) [reply src/pages/api/changeCategory.ts]: done

**Review by RealBirdMan91 — CHANGES_REQUESTED** [review]
looks good =)

But you don't need that one 

```TS
else {
    response
      .status(404)
      .send(
        `Invalid method in a front-end-script, need a PATCH to update the category, request method was ${request.method}`
      );
  }
```

**Comment by amyria3** [comment]
done

**Review by RealBirdMan91 — APPROVED** [review]

**Pull request merged**